ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -117,7 +124,43 @@ public override void OnCollectExtraUpdates(UIViewOperationQueue uiViewOperationQ
 
             if (_inline != null)
             {
-                uiViewOperationQueue.EnqueueUpdateExtraData(ReactTag, _inline);
+                Action<Border> action = (border) => 
```

Comment:
Very sorry about this, but I think using the action may actually have different semantics.  I believe that this Action will create a closure type with a reference to the "this" object, so when the action is invoked, it will use whatever the latest value of the fields are, rather than what the fields were at the time OnCollectExtraUpdates was called.  I think this could lead to some interesting bugs.